### PR TITLE
[FIX] fix order funded amount calculation

### PR DESCRIPTION
**Pull request by boxbag**

### CODE REVIEW

**Comment by boxbag** [comment]
Hold off on reviewing, i'm going to add more changes

**Comment by boxbag** [comment]
Ready for review

File: src/js/ripple/orderbook.js [hunk 1]
```diff
@@ -922,41 +927,59 @@ OrderBook.prototype.insertOffer = function(node) {
...
+ * @param {Object} node - Offer node
+ * @param {Boolean} isOfferCancel - whether node came from an OfferCancel transaction
+ */
+
+OrderBook.prototype.deleteOffer = function(node, isOfferCancel) {
+  if (this._remote.trace) {
+    log.info('deleting offer', this._key, node.fields);
+  }
+
+  for (var i=0; i<this._offers.length; i++) {
```
Comment (wltsmrz) [inline src/js/ripple/orderbook.js]: Are there lodash methods you can use here to keep things consistent?

**Comment by wltsmrz** [comment]
LGTM

File: src/js/ripple/orderbook.js (hunk 1 above)
Comment (boxbag) [inline src/js/ripple/orderbook.js]: Unfortunately we're doing concurrent modification here so I didn't want to use any iteration HOFs

Comment (wltsmrz) [inline src/js/ripple/orderbook.js]: Fair enough

**Pull request merged**